File: frontend/fix_urls.js
const fs = require('fs');
const file = 'c:/Users/shaha/OneDrive/Desktop/AI-Skill-Assessment-Platform/frontend/src/pages/Dashboard/RecruiterDashboard.jsx';
let content = fs.readFileSync(file, 'utf8');
content = content.replace(/`\\\/recruiter\//g, '`${API_URL}/recruiter/');
fs.writeFileSync(file, content);
console.log('Fixed URLs');

File: frontend/fix_urls.py
import os
import re

file_path = 'c:/Users/shaha/OneDrive/Desktop/AI-Skill-Assessment-Platform/frontend/src/pages/Dashboard/RecruiterDashboard.jsx'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Replace any variant of `\/recruiter/ or `/recruiter/
content = re.sub(r"`\\?/recruiter/", r"`${API_URL}/recruiter/", content)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print("Done resetting URLs!")
